cosmoz-autocomplete.js

```diff
@@ -665,6 +665,10 @@
 			this._errorMessage = '';
 			// passed all the above
 			return true;
+		},
+
+		_getSelectedItemTitle: function (item, prop) {
```

Comment:
Polymer lint complains that `get` is not a function when doing that in the attribute data binding.